--- libdvb-0.5.0/dvb-mpegtools/Makefile.fix	2003-11-06 19:26:05.946692384 +0100
+++ libdvb-0.5.0/dvb-mpegtools/Makefile	2003-11-06 19:30:49.633565424 +0100
@@ -62,7 +62,7 @@
 	install -m 755 dvb-mpegtools_main $(PREFIX)/bin/
 	install -m 755 dvbaudio $(PREFIX)/bin/
 	for f in $(LINKS); do\
-		ln -sf $(PREFIX)/bin/dvb-mpegtools_main $(PREFIX)/bin/$$f ;\
+		cp -a $$f $(PREFIX)/bin/ ;\
 	done
 
 uninstall:
