--- libdvb-0.5.5.1/dvb-mpegtools/show_dvb_image.c.orig	2005-03-22 13:23:25.000000000 +0000
+++ libdvb-0.5.5.1/dvb-mpegtools/show_dvb_image.c	2005-12-30 09:22:02.005433224 +0000
@@ -71,7 +71,7 @@
 		return;
 	}
 
-	printf("read: %d bytes\n",read(filefd,sp.iFrame,sp.size));
+	printf("read: %Zd bytes\n",read(filefd,sp.iFrame,sp.size));
 	videoStillPicture(fd,&sp);
 
 	sleep(ti);
--- libdvb-0.5.5.1/libdvbmpeg/ringbuffy.c.orig	2005-03-22 13:23:25.000000000 +0000
+++ libdvb-0.5.5.1/libdvbmpeg/ringbuffy.c	2005-12-30 09:04:04.393255176 +0000
@@ -19,6 +19,7 @@
 */
 
 #include "ringbuffy.h"
+#include <string.h>
 
 int ring_init (ringbuffy *rbuf, int size)
 {
--- libdvb-0.5.5.1/libdvbmpeg/transform.c.orig	2005-03-22 13:23:25.000000000 +0000
+++ libdvb-0.5.5.1/libdvbmpeg/transform.c	2005-12-30 09:18:31.636414176 +0000
@@ -2556,7 +2556,7 @@
 					      get_vinfo( pay, l,&p->vi,1)+1) >0
 						){
 						vpts = trans_pts_dts(sb+9);
-						vpos = (int)(l-count+pay);
+						vpos = (int)(l-count+(long)pay);
 						printf("vpts : %fs\n",
 						       vpts/90000.);
 					}
@@ -2573,7 +2573,7 @@
 					      get_ac3info( pay, l,&p->ai,1)+1) >0
 						){
 						apts = trans_pts_dts(sb+9);
-						apos = (int)(l-count+pay);
+						apos = (int)(l-count+(long)pay);
 						printf("apts : %fs\n",
 						       apts/90000.);
 					}
--- libdvb-0.5.5.1/sample_progs/cam_menu.cc.orig	2005-03-22 13:23:25.000000000 +0000
+++ libdvb-0.5.5.1/sample_progs/cam_menu.cc	2005-12-30 09:20:16.720438976 +0000
@@ -46,7 +46,7 @@
 
 void byebye(int i, void *f)
 {
-	int fd = (int) f;
+	int fd = (long) f;
 
 	CamPacket cp(fd, CMENU_CLOSE, 0, NULL);
 	cp.psend();
--- libdvb-0.5.5.1.orig/sample_progs/cam_menu.hh    2008-01-28 10:23:53.000000000 +0100
+++ libdvb-0.5.5.1/sample_progs/cam_menu.hh         2008-01-28 10:22:55.000000000 +0100
@@ -2,6 +2,7 @@
  * cam_menu.hh
  *
  */
+#include <cstring>
 #include <sys/types.h>
 #include <sys/socket.h>


